trafficFlowTests.py

```diff
@@ -85,123 +87,115 @@ def run_tests(self, servers, clients, monitors, duration: int):
             tasks.stop()
 
         for tasks in servers + clients + monitors:
-            tasks.output()
+            tasks.output(tft_aggregate_output)
+
+        return tft_aggregate_output
 
-    def create_log_paths_from_tests(self, tests: dict):     
+    def _create_log_paths_from_tests(self, tests: dict):     
         if "logs" in tests:
             self.log_path = Path(tests['logs'])
-        self.log_path = self.log_path
-        logger.info(f"Logs will be written to {self.log_path}")
-        if self.log_path.is_dir():
-            shutil.rmtree(self.log_path)
-        
-        # Create directory for each connection / instance to store run results
-        for connections in tests['connections']:
-            for index in range(connections['instances']):
-                path=Path(f"{self.log_path}/{connections['name']}-{index}")
-                logger.info(f"Creating dir {path}")
-                path.mkdir(parents=True)
-        
+        self.log_path.mkdir(parents=True, exist_ok=True)
+        timestamp = datetime.datetime.now().strftime("%Y-%m-%d-%H-%M-%S")
+        self.log_file = self.log_path / f"{timestamp}.json"
+        logger.info(f"Logs will be written to {self.log_file}")
+
+    def _dump_result_to_log(self):
+        # Dump test outputs into log file
+        log = self.log_file
+        json_out = {"runs": []}
+        for out in self.tft_output:
+            json_out["runs"].append(
+                out.dump_to_dict()
+            )
+        with open(log, "w") as output_file:
+            json.dump(json_out, output_file)
+
+    def evaluate_run_success(self) -> bool:
+        all_passing = True
```

Comment:
what is this variable for? It doesn't look like it's being used.

Reply:
Aahh it is being used, but not meaningfully with the new change. I will remove this thanks